### PR TITLE
Integrate CC Vocabulary components into CC Search

### PR DESCRIPTION
**Pull request by dhruvkb**
This is a long term PR that tracks the integration of CC Vocabulary into CC Search, component by component. This PR will enable real-time tracking as more and more components are developed and refined in CC Vocabulary and then plugged into the CC Search UI.
<details>
<summary>
Developer Certificate of Origin Version 1.1
</summary>
<p>
Copyright (C) 2004, 2006 The Linux Foundation and its contributors. 1 Letterman Drive Suite D4700 San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this license document, but changing it is not allowed.

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I have the right to submit it under the open source license indicated in the file; or

(b) The contribution is based upon previous work that, to the best of my knowledge, is covered under an appropriate open source license and I have the right under that license to submit that work with modifications, whether created in whole or in part by me, under the same open source license (unless I am permitted to submit under a different license), as indicated in the file; or

(c) The contribution was provided directly to me by some other person who certified (a), (b) or (c) and I have not modified it.

(d) I understand and agree that this project and the contribution are public and that a record of the contribution (including all personal information I submit with it, including my sign-off) is maintained indefinitely and may be redistributed consistent with this project or the open source license(s) involved.
</p>
</details>

### CODE REVIEW

**Comment by dhruvkb** [comment]
**Screenshots of the CC Vocabulary header on CC Search**:
Desktop:
image
Phone:
image

**Comment by dhruvkb** [comment]
**Screenshots of the CC Vocabulary header on CC Search:**
image
Note that the fonts are being messed up by Foundation CSS. The end goal is to rid CC Search of Foundation when CC Vocabulary subsumes all its functionality.

**Comment by dhruvkb** [comment]
@brenoferreira It'd be very helpful if you could have clone this branch and provide feedback on the redesign. I had to take a few liberties along the way which I'm not sure about.

File: src/i18n.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      if (matched && matched.length > 1) {
+        const locale = matched[1];
+        messages[locale] = locales(key);
+      }
+    });
+  return messages;
+}
+
+export default new VueI18n({
+  locale: process.env.VUE_APP_I18N_LOCALE || 'en',
```
Comment (brenoferreira) [inline src/i18n.js]: If you're loading the locales from an env variable, it's always going to be in one language for all users, isn't it? This should come from the browser information, and in the case of server rendering, there should be something in the request headers that indicate the client's preferred language

Comment (dhruvkb) [reply src/i18n.js]: This option only sets the default locale when building, the chosen locale has to be saved in a browser cookie or `localStorage` for permanence across sessions.

Comment (dhruvkb) [reply src/i18n.js]: Will incorporate this in an update! :+1:

File: build/vue-loader.conf.js [hunk 2]
```diff
@@ -7,10 +7,15 @@ const sourceMapEnabled = isProduction
   : config.dev.cssSourceMap
 
 module.exports = {
-  loaders: utils.cssLoaders({
-    sourceMap: sourceMapEnabled,
-    extract: isProduction
-  }),
+  loaders: Object.assign({},
```
Comment (xumisyou) [inline build/vue-loader.conf.js]: h

**Comment by annatuma** [comment]
@dhruvkb please let us know what the status of this ticket is, if you have any blockers to moving this forward, etc. We'd love to help get this to completion.

**Comment by dhruvkb** [comment]
@annatuma this has been blocked because of a presentational conflict with Foundation because Vocabulary typography and spacing is being overridden by the site-wide Foundation CSS. Also I've not been able to manage time between Vocabulary stewardship and working on this. If this is still high on priority, I'll try to make time for this.

**Comment by brenoferreira** [comment]
This is out of date and not relevant anymore

**Pull request closed without merging**